# HG changeset patch
# User Rik <rik@octave.org>
# Date 1373685483 25200
#      Fri Jul 12 20:18:03 2013 -0700
# Node ID e424cdb2ef968db42dfe36417510ea9388896e5b
# Parent  52c1b832d98eb4b2ff5207e48430f56b46af29c5
pol2cart.m: Correct typo in error() message.

* scripts/general/pol2cart.m: Correctly spell name of function in error()
message.

diff --git a/scripts/general/pol2cart.m b/scripts/general/pol2cart.m
--- a/scripts/general/pol2cart.m
+++ b/scripts/general/pol2cart.m
@@ -50,17 +50,17 @@ function [x, y, z] = pol2cart (theta, r,
       if (columns (theta) == 3)
         z = theta(:,3);
       else
         z = [];
       endif
       r = theta(:,2);
       theta = theta(:,1);
     else
-      error ("pol2car: matrix input must have 2 or 3 columns [THETA, R (, Z)]");
+      error ("pol2cart: matrix input must have 2 or 3 columns [THETA, R (, Z)]");
     endif
   elseif (nargin == 2)
     if (! ((ismatrix (theta) && ismatrix (r))
             && (size_equal (theta, r) || isscalar (theta) || isscalar (r))))
       error ("pol2cart: arguments must be matrices of same size, or scalar");
     endif
   elseif (nargin == 3)
     if (! ((ismatrix (theta) && ismatrix (r) && ismatrix (z))
